src/main/java/org/snakeyaml/engine/v2/comments/CommentEventsCollector.kt

```diff
@@ -153,11 +153,10 @@ class CommentEventsCollector {
         }
     }
 
-    val isEmpty: Boolean
-        /**
-         * Test if the collector contains any collected events.
-         *
-         * @return `true` if it does; Otherwise, `false`
-         */
-        get() = commentLineList.isEmpty()
+    /**
+     * Test if the collector contains any collected events.
+     *
+     * @return `true` if it does; Otherwise, `false`
+     */
+    fun isEmpty(): Boolean = commentLineList.isEmpty()
```

Comment:
renamed to match the JavaBeans naming conventions (properties shouldn't start with `is`), because I think it's nice to be consistent.